File: src/main/java/github/zmilla93/gui/setup/StartSetupPanel.java
package github.zmilla93.gui.setup;

import github.zmilla93.core.utility.ZUtil;

import javax.swing.*;
import java.awt.*;

public class StartSetupPanel extends JPanel {

    public StartSetupPanel() {
        setLayout(new GridBagLayout());
        GridBagConstraints gc = ZUtil.getGC();
        add(new JLabel("Welcome to SlimTrade!"), gc);
        gc.gridy++;
        add(new JLabel("Let's do some quick setup."), gc);
        gc.gridy++;
    }

}

File: src/main/java/github/zmilla93/gui/setup/StartSetupPanel.kt
package github.zmilla93.gui.setup

import github.zmilla93.core.utility.ZUtil.getGC
import java.awt.GridBagLayout
import javax.swing.JLabel
import javax.swing.JPanel

class StartSetupPanel : JPanel() {

    init {
        setLayout(GridBagLayout())
        val gc = getGC()
        add(JLabel("Welcome to SlimTrade!"), gc)
        gc.gridy++
        add(JLabel("Let's do some quick setup."), gc)
        gc.gridy++
    }

}
